### PR TITLE
Fixes #31470 - configure errata_ids as a required param

### PR DESCRIPTION
**Pull request by chris1984**
Screenshot of updated API docs:

https://nimbusweb.me/s/share/4958108/kkcp3149abi7yr44d2nz

### CODE REVIEW

**Comment by chris1984** [comment]
[test katello]

File: app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb [hunk 1]
```diff
@@ -267,7 +267,7 @@ def system_purpose
     api :POST, "/hosts/bulk/available_incremental_updates", N_("Given a set of hosts and errata, lists the content view versions" \
                                                                  " and environments that need updating.")
     param_group :bulk_params
-    param :errata_ids, Array, :desc => N_("List of Errata ids")
+    param :errata_ids, Array, :desc => N_("List of Errata ids"), :required => true
```
Comment (jturel) [inline app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb:270]: Does the API give an error if the errata_ids arent specified? Might be a good addition if not

Comment (chris1984) [reply app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb:270]: @jturel updated and included a test to confirm we get an error when not sending the param :)

File: test/controllers/api/v2/hosts_bulk_actions_controller_test.rb [hunk 2]
```diff
@@ -325,6 +325,11 @@ def test_available_incremental_updates
       assert_response :success
     end
 
+    def test_incremental_updates_no_ids
+      post :available_incremental_updates, params: { :included => {:ids => [@host1.id]}, :organization_id => @org.id }
+      assert_response :error
```
Comment (jturel) [inline test/controllers/api/v2/hosts_bulk_actions_controller_test.rb:330]: assert_response :bad_request

File: app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb [hunk 3]
```diff
@@ -267,8 +267,9 @@ def system_purpose
...
     def available_incremental_updates
+      fail _("errata_ids is a required paramater") unless params[:errata_ids].present?
```
Comment (jturel) [inline app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb:272]: This would be the right time to raise HttpErrors::BadRequest !

Comment (jturel) [reply app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb:272]: s/paramater/parameter

**Comment by chris1984** [comment]
@jturel thanks for the review, updated.

**Review by jturel — APPROVED** [review]
APJ

**Pull request merged**